dref/models.py

```diff
@@ -518,6 +517,16 @@ class Meta:
         verbose_name_plural = _('drefs')
 
     def save(self, *args, **kwargs):
+        if self.budget_file:
```

Comment:
We also need to check if this has changed. Else this will be triggered for every save.